tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/upgrade/animated/InitConfigureProjectPage.java

```diff
@@ -847,6 +847,23 @@ private void deleteEclipseConfigFiles( File project )
         }
     }
 
+    private void deleteServiceBuilderJarFile( IProject project, IProgressMonitor monitor )
+    {
+        try
+        {
+            IFile serviceJarFile = project.getFile( "docroot/WEB-INF/lib/" + project.getName() + "-service.jar" );
```

Comment:
We shouldn't hardcode "docroot", but instead use the "IWebProject" get defaultDocrootFolder and then look for

"WEB-INF/lib" folder after that.